addons/binding/org.openhab.binding.hdanywhere/OSGI-INF/HDanywhereHandlerFactory.xml

```diff
@@ -10,11 +10,13 @@
 
 -->
 <scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.openhab.binding.hdanywhere.internal.HDanywhereHandlerFactory">
-
+  
    <implementation class="org.openhab.binding.hdanywhere.internal.HDanywhereHandlerFactory"/>
-
+   <reference bind="setItemChannelLinkRegistry" cardinality="1..1" interface="org.eclipse.smarthome.core.thing.link.ItemChannelLinkRegistry" name="ItemChannelLinkRegistry" policy="static" unbind="unsetItemChannelLinkRegistry"/>
```

Comment:
No, it is not, see [here](https://www.eclipse.org/smarthome/documentation/development/bindings/dependencies.html).
I guess Karel wants to work around the fact that https://github.com/eclipse/smarthome/issues/595 isn't yet implemented.